cmd/kube-mgmt/main.go

```diff
@@ -106,6 +113,27 @@ func run(params *params) {
 		params.opaAuth = strings.Split(string(file), "\n")[0]
 	}
 
+	if params.opaAllowInsecure || params.opaCAFile != "" {
```

Comment:
Would it make sense to make these mutually exclusive? I.e. if a cert is provided, would you still want to allow insecure?

Reply:
I don't think there is a logical reason to allow both. I'll make them mutually exclusive.